src/main/java/sg/ncl/MainController.java

```diff
@@ -132,9 +149,26 @@ public void futureplanDownload(HttpServletResponse response) {
             IOUtils.copy(is, response.getOutputStream());
             response.flushBuffer();
             is.close();
+        } catch (Exception ex) {
+            logger.info("Error writing file to output stream.");
+            throw new FuturePlanDownloadException("IOError writing file to output stream");
+        }
+    }
+
+    @RequestMapping(value="/OrderForm_v1/download", method=RequestMethod.GET)
+    public void OrderForm_v1Download(HttpServletResponse response) throws Exception {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00112)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00100)